bark/generation.py

```diff
@@ -156,29 +162,12 @@ def _grab_best_device(use_gpu=True):
     return device
 
 
-S3_BUCKET_PATH_RE = r"s3\:\/\/(.+?)\/"
-
-
-def _parse_s3_filepath(s3_filepath):
-    bucket_name = re.search(S3_BUCKET_PATH_RE, s3_filepath).group(1)
-    rel_s3_filepath = re.sub(S3_BUCKET_PATH_RE, "", s3_filepath)
-    return bucket_name, rel_s3_filepath
-
-
-def _download(from_s3_path, to_local_path):
+def _download(from_hf_path, file_name, to_local_path):
     os.makedirs(CACHE_DIR, exist_ok=True)
-    response = requests.get(from_s3_path, stream=True)
-    total_size_in_bytes = int(response.headers.get("content-length", 0))
-    block_size = 1024
-    progress_bar = tqdm.tqdm(total=total_size_in_bytes, unit="iB", unit_scale=True)
-    with open(to_local_path, "wb") as file:
-        for data in response.iter_content(block_size):
-            progress_bar.update(len(data))
-            file.write(data)
-    progress_bar.close()
-    if total_size_in_bytes != 0 and progress_bar.n != total_size_in_bytes:
-        raise ValueError("ERROR, something went wrong")
-
+    destination_file_name = to_local_path.split("/")[-1]
+    file_dir = CACHE_DIR
+    hf_hub_download(repo_id=from_hf_path, filename=file_name, local_dir=file_dir)
+    os.replace(f"{CACHE_DIR}/{file_name}", to_local_path)
```

Comment:
You need to be careful about Windows users here:

```suggestion
    hf_hub_download(repo_id=from_hf_path, filename=file_name, local_dir=CACHE_DIR)
    os.replace(os.path.join(CACHE_DIR, file_name), to_local_path)
```